### PR TITLE
Added a command to grab a random xkcd comic

### PR DESCRIPTION
**Pull request by paynejordand**
I'm bored and don't want to work on 3500 so I added this. It's pretty simple. There is a permanent redirect url that redirects to a random comic.

### CODE REVIEW

File: bot/cogs/random_cog/random_cog.py [hunk 1]
```diff
@@ -171,5 +174,22 @@ async def raffle(self, ctx, time: typing.Optional[Duration] = 5, *, reason):
                     embed = discord.Embed(title = 'RAFFLE', color=Colors.ClemsonOrange, description = description)
                     await msg.edit(embed = embed)
 
+    @commands.command(aliases=['xkcd'])
+    async def relevant(self, ctx):
```
Comment (Jay-Madden) [inline bot/cogs/random_cog/random_cog.py:178]: lol this is a funny command name

File: bot/cogs/random_cog/random_cog.py [hunk 2]
```diff
@@ -171,5 +174,22 @@ async def raffle(self, ctx, time: typing.Optional[Duration] = 5, *, reason):
...
+        """
+        Generates a possibly relevant xkcd.
+        https://c.xkcd.com/random/comic/ is a random comic from the xkcd catalogue
+        """
+        async with aiohttp.ClientSession() as session:
+            async with await session.get(url="https://c.xkcd.com/random/comic/") as resp:
+                if(resp.status == 200):
+                    await ctx.send(resp.url)
```
Comment (Jay-Madden) [inline bot/cogs/random_cog/random_cog.py:186]: should the comic itself be deletable?

File: bot/cogs/random_cog/random_cog.py [hunk 3]
```diff
@@ -171,5 +174,22 @@ async def raffle(self, ctx, time: typing.Optional[Duration] = 5, *, reason):
                     embed = discord.Embed(title = 'RAFFLE', color=Colors.ClemsonOrange, description = description)
                     await msg.edit(embed = embed)
 
+    @commands.command(aliases=['xkcd'])
+    async def relevant(self, ctx):
+        """
+        Generates a possibly relevant xkcd.
+        https://c.xkcd.com/random/comic/ is a random comic from the xkcd catalogue
+        """
+        async with aiohttp.ClientSession() as session:
```
Comment (Jay-Madden) [inline bot/cogs/random_cog/random_cog.py:183]: good use of async here

**Review by Jay-Madden — CHANGES_REQUESTED** [review]

File: bot/cogs/random_cog/random_cog.py (hunk 3 above)
Comment (paynejordand) [reply bot/cogs/random_cog/random_cog.py:183]: Straight from the docs lmao

File: bot/cogs/random_cog/random_cog.py (hunk 2 above)
Comment (paynejordand) [reply bot/cogs/random_cog/random_cog.py:186]: The embeds are typically pretty small but yeah, I can make it deleteable

**Review by Jay-Madden — APPROVED** [review]
lgtm, i love this idea

**Pull request merged**